src/vaults/SyncDepositVault.sol

```diff
@@ -30,7 +30,7 @@ contract SyncDepositVault is BaseSyncDepositVault, BaseAsyncRedeemVault {
         ISyncDepositManager syncDepositManager_,
         IAsyncRedeemManager asyncRedeemManager_
     )
-        BaseVault(poolId_, scId_, asset_, token_, root_, syncDepositManager_)
+        BaseVault(poolId_, scId_, asset_, token_, root_, IBaseRequestManager(address(asyncRedeemManager_)))
```

Comment:
This means the main manager is the async one in the `BaseVault`, right?

Reply:
Since that is the only one inheriting from `BaseRequestManager` now